components/fl-orchestrator/Makefile

```
@@ -4,6 +4,9 @@ MAKEVAR_REGISTRY?=ghcr.io/katulu-io/fl-suite
 MAKEVAR_VERSION?=0.0.0-dev-$(shell git log -n 1 --pretty=format:%h)
 CONTAINER_TAG = $(shell echo "${MAKEVAR_VERSION}" | sed "s/\+[0-9a-zA-Z-]*//")
 
+PROTOC_VERSION=21.4
+GOBIN = $(shell pwd)/bin
```

nit: GNU make already comes with a variable called `$CURDIR` that is set tot the current working directory where the Makefile is:

```suggestion
GOBIN = $(CURDIR)/bin
```

---

Awesome, didn't know about that!